src/components/Card.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Card from './Card';
+import { shallow } from 'enzyme';
+
+describe('Card', () => {
+  test('shallow mount', () => {
```

Comment:
It could also be helpful to have additional tests for `Card` which check the snapshot if the component is rendered with a specific set of props.

Right now we're providing the `id`, `index`, and `deleteCardCallback` props, but none of these impact on the rendering. I think we could have further tests which provide the `text` and `emoji` props (one for each, and a third for when both are provided), as those props do impact the visual result.